doc/source/conf_mat.rst

```diff
@@ -0,0 +1,249 @@
+Confusion matrix
+================
+
+Import ``ConfusionMatrix``
+
+.. code-block:: python
+
+    from pandas_ml.confusion_matrix import ConfusionMatrix
+
+Define actual values (``y_actu``) and predicted values (``y_pred``)
```

Comment:
Can you use ``y_true`` ash the same as ``scikit-learn``?

- http://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html